python/Glacier2/callback/README.md

```diff
@@ -1,9 +1,9 @@
 # Glacier2 Callback
 
-This demo shows how to write a client that establishes a session with a Glacier2 router. It also shows how to implement
-callbacks in this client.
+This demo shows how to write a client that establishes a session with a Glacier2 router.  
```

Comment:
Not sure if it's just rendering, is there extra whitespace at the end of this line?